public/index.html

```diff
@@ -7,12 +7,15 @@
   <title>Are you alive?</title>
```

Comment:
I can't figure out how to add it back without it glitching out for me. I might need your help with the CSS